swarm/src/handler.rs

```diff
@@ -112,8 +113,10 @@ pub trait ConnectionHandler: Send + 'static {
     /// The outbound upgrade for the protocol(s) used by the handler.
     type OutboundProtocol: OutboundUpgradeSend;
     /// The type of additional information returned from `listen_protocol`.
+    #[deprecated]
     type InboundOpenInfo: Send + 'static;
     /// The type of additional information passed to an `OutboundSubstreamRequest`.
+    #[deprecated]
     type OutboundOpenInfo: Send + 'static;
```

Comment:
Please add some comment here, e.g. `deprecated = "Track data in ConnectionHandler instead."`